stable/artifactory/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [8.3.2] - Dec 10, 2019
+* Added velero/rustic backup options to helm
```

Comment:
Is it rustic or restic?

Reply:
Restic, sorry about that. I'll fix this.